packages/react-app/src/components/global/Footer/index.tsx

```diff
@@ -12,24 +12,27 @@ const Footer = (): JSX.Element => {
 			px={8}
 			py={10}
 			width="full"
-			justifyContent="space-between"
+			wrap="wrap"
+			alignItems="center"
+			justifyContent={['center', 'space-between']}
 		>
-			<Box>
-				{/* <Text fontSize="xs" color="grey">
-          Mirror Substack Discord Twitter Github
-        </Text> */}
+			<Flex>
 				<HStack>
 					<AccessibleLink href={feedbackUrl} isExternal={true}>
 						<ColorModeButton>Give us Feedback</ColorModeButton>
 					</AccessibleLink>
-
 					<AccessibleLink href={discordSupportChannelUrl} isExternal={true}>
 						<ColorModeButton>Need Help?</ColorModeButton>
 					</AccessibleLink>
 				</HStack>
-			</Box>
-
-			<Text>&copy; {new Date().getFullYear()} Bankless DAO</Text>
+			</Flex>
+			<Text
+				my={['5', '5', '0']}
```

Comment:
Haven't been able to find documentation on this prop. It's inferable, but would appreciate some clarity